### PR TITLE
Update built-in Spark version to 3.0.2

### PR DESCRIPTION
**Pull request by ulysses-you**
ulysses-you [Closes #356](https://github.com/yaooqinn/kyuubi/pull/356)    Target Issue Test Plan [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _Why are the changes needed?_

Spark 3.0.2 has been released now.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/tools/distribution.md [hunk 1]
```diff
@@ -28,4 +28,4 @@ For instance,
 ./build/dist --name custom-name --tgz
 ```
 
-This results a Kyuubi distribution named `kyuubi-{version}-bin-custom-name-spark-3.0.1.tar.gz` for you. 
+This results a Kyuubi distribution named `kyuubi-{version}-bin-custom-name.tar.gz` for you.
```
Comment (ulysses-you) [inline docs/tools/distribution.md:31]: We don't give the spark version if user give a special name.
```
if [[ "$NAME" == "none" ]]; then
  NAME="spark-"$SPARK_VERSION
fi

```

File: bin/load-kyuubi-env.sh [hunk 2]
```diff
@@ -66,7 +66,7 @@ fi
 SPARK_BUILTIN="${KYUUBI_HOME}/externals/spark-$SPARK_VERSION_BUILD-bin-hadoop${HADOOP_VERSION_BUILD:0:3}$HIVE_VERSION_SUFFIX"
 
 if [[ ! -d ${SPARK_BUILTIN} ]]; then
-  SPARK_BUILTIN="${KYUUBI_HOME}/externals/kyuubi-download/target/spark-3.0.1-bin-hadoop2.7"
+  SPARK_BUILTIN="${KYUUBI_HOME}/externals/kyuubi-download/target/spark-$SPARK_VERSION_BUILD-bin-hadoop${HADOOP_VERSION_BUILD:0:3}"
```
Comment (ulysses-you) [inline bin/load-kyuubi-env.sh:69]: Just clean the hard code. Hope it doesn't hurt something.

**Comment by ulysses-you** [comment]
cc @yaooqinn

File: bin/load-kyuubi-env.sh (hunk 2 above)
Comment (yaooqinn) [reply bin/load-kyuubi-env.sh:69]: we do not have release file in source code

Comment (yaooqinn) [reply bin/load-kyuubi-env.sh:69]: https://github.com/yaooqinn/kyuubi/blob/master/.travis.yml#L51

we can evaluate spark and hadoop compile version w/ maven.

Comment (ulysses-you) [reply bin/load-kyuubi-env.sh:77]: Updated, from maven.

**Review by yaooqinn — APPROVED** [review]

**Comment by ulysses-you** [comment]
merging to master ! FYI @yaooqinn

**Pull request closed without merging**